src/editors/components/EditorFooter/index.jsx

```diff
@@ -9,24 +9,38 @@ import {
   ModalDialog,
   Toast,
 } from '@edx/paragon';
+import { sendTrackEvent } from '@edx/frontend-platform/analytics';
 import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import { nullMethod, saveBlock, navigateCallback } from '../../hooks';
 
 import { RequestKeys } from '../../data/constants/requests';
 import { selectors, thunkActions } from '../../data/redux';
 
 import messages from './messages';
+import analytics from '../../analytics';
 import * as module from '.';
 
-export const handleSaveClicked = (props) => () => saveBlock(props);
-export const handleCancelClicked = ({ returnUrl }) => navigateCallback(returnUrl);
+export const handleSaveClicked = ({ saveProps, returnAnalytics }) => () => {
+  console.log('test save', returnAnalytics);
+  console.log('test save', sendTrackEvent)
```

Comment:
should remove

Reply:
Also, these should be added into saveBlock and and navigateCallback in the editor hooks.js file